sympy/functions/elementary/complexes.py

```diff
@@ -466,7 +466,7 @@ def eval(cls, arg):
                     if base.func is cls and exponent is S.NegativeOne:
                         return arg
                     return Abs(base)**exponent
-                if base.is_positive == True:
+                if base.is_positive is True:
```

Comment:
I'm not sure what the best way to make this kind of comparison is.
base.is_positive == True?
base.is_positive is True?
base.is_positive?

This should probably be discussed on the mailing list.